lib/rmt/cli/custom_repos.rb

```diff
@@ -65,15 +65,15 @@ def remove(repository_id)
       return
     end
 
-    repository_service.remove_repository(repository)
+    Repository.remove(repository)
```

Comment:
Not sure how to put it, but something feels a bit wrong here: First you check here if the repo is custom. Then you call a method `remove` which does the check again. Which is not obvious by it's name... Who will remember in 3 month that remove does sth different than destroy? After all these checks you could just use the destroy method here, I think. Or if you want a more save method on the model, which still does the check itself, you might want to call it without repeating the checks here. And instead, just make a conditional that outputs the warning when the returned value of the method was false. Something like that could remove this repeated checking.